src/modules/edc-demo/components/error-state/error-state.component.html

```diff
@@ -0,0 +1,6 @@
+<mat-icon style="font-size: 80px; width: 80px; height: 80px" color="warn"
+  >error
+</mat-icon>
+<div class="class=text-center mt-[20px] w-full">
```

Comment:
While trying it out I noticed the class mt-[20px] does not exist in style.scss. It was never added or was accidentally removed in the main branch, so right now this element has no margin top.

I'd recommend mt-[10]px.

Also again we have the issue with `class="class=` making the text-center class not work.

When you compare this message to the dashboard, it's missing the classes "text-warn uppercase"